app/src/common/chromium/com/igalia/wolvic/browser/api/impl/SessionImpl.java

```diff
@@ -54,7 +59,7 @@ public SessionImpl(@Nullable WSessionSettings settings) {
 
     private void init() {
         mTextInput = new TextInputImpl(this);
-        mPanZoomCrontroller = new PanZoomCrontrollerImpl(this);
+        mPanZoomController = new PanZoomControllerImpl(this);
```

Comment:
Same here, the panZoomController typo can be a separate commit, it's unrelated to the rest of the changes

Reply:
Removed and fixed this typo issue by Zakhar.